frontend/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
you might want to replace this template README with some basic instructions on how to run/build the frontend

Reply:
Yes, that was an honest oversight. I'll add the necessary instructions right away.

Reply:
I've made a new push and the README has been updated.